### PR TITLE
Jj/arm bitcode generation

### PR DESCRIPTION
**Pull request by JuliaJakubcova**

### CODE REVIEW

File: samples/sample-bugsnag/shared/build.gradle.kts [hunk 1]
```diff
@@ -76,7 +76,6 @@ kotlin {
             export("co.touchlab:kermit:${KERMIT_VERSION}")
             export("co.touchlab:kermit-bugsnag:${KERMIT_VERSION}")
             export(libs.crashkios.bugsnag)
-            isStatic = false
```
Comment (samhill303) [inline samples/sample-bugsnag/shared/build.gradle.kts:79]: Lets leave in the setting but set it to true

**Review by samhill303 — APPROVED** [review]

File: gradle/configure-crash-logger.gradle [hunk 2]
```diff
@@ -27,24 +27,24 @@ kotlin {
         macosArm64(),
         iosX64(),
         iosArm64(),
-        iosArm32(),
+//        iosArm32(),
```
Comment (russhwolf) [inline gradle/configure-crash-logger.gradle:30]: I don't think we should be disabling 32-bit targets

Comment (kpgalligan) [reply gradle/configure-crash-logger.gradle:30]: That was me. Rick doesn't publish them. We'll have to resolve that at some point soon, but didn't want that to be a blocker.

**Comment by samhill303** [comment]
Closing as I included this in #309

**Pull request closed without merging**